### DISCUSSION TITLE
Creating volumes

### DISCUSSION THREAD

**Discussion by Nico99-GU** [biigle/core#1015] [Q&A] [answered]
Hi,
I'm having problems creating a volume. I already have a project called "Skagerrak_SLU" in which I created a volume "4R_98SLU". I wanted to add a new video in a new volume in that same project. But for some reason I canno't change the directory/file source and BIIGLE keeps using the one I used for my first volume. I had to reorder my folders few days ago. I thought this might be the reason. That for within a project, all files need to be in the same folder. So I decided to create a new project where I would have all my videos/images in the same place. But the problem is the same. I canno't select the folder I want where my video is.
BIIGLE tells me that This is a pending volume that you did not finish before. I don't know if that's linked.

How can I select another folder ?
Could you help me?
Please tell me if you can0t understand and I will explain in another way.
Thank you
Nicola Philbert
<img width="509" alt="Capture d’écran, probleme de volume" src="https://github.com/user-attachments/assets/fd5dd57c-6e41-4e71-a6fd-e89cbef3074e" />

**Comment by mzur** [comment, marked as answer]
Hi Nicola, only images or videos of a single volume have to be in the same folder. If you have different volumes, you can store the files in different folders, too. The pending volume means that you started to create a new volume before but didn't finish. Maybe you started to create a video volume but actually want to create an image volume? In your current pending volume view, only video files will be displayed. If you want to create an image volume instead, click the "cancel" button there and restart the process with a new image volume.

**Reply by Nico99-GU** [reply]
Hi,
I wanted to create a new video volume. I don't know how to explain this but I don't understand how I can choose the file where my MP4 videos are. 
Under "Choose a file source", a path directory is already selected (see Image) and I don't know how to change it. This path directory is the one used for my first volume.
I tried to cancel and start again but it doesn't work.

**Reply by mzur** [reply]
You can select a different directory with a click. You have to select a directory that directly contains the video files. If you select a parent directory that contains subdirectories with files, it doesn't work.

**Reply by Nico99-GU** [reply]
I'm really sorry to bother with my questions. But it really seems like I can't change anything. The path directory showed in the picture above is the only option given to me. Also, this path no longer exists in my computer. A few days ago, I reordered my folders. Moving videos to new folders on my desktop. 
I asked a colleague how it was for her and I think my interface has a problem. 
Is the path created from the directories and subdirectories that we choose when creating the storage?
P.S. My videos are in MP4 format.

**Reply by mzur** [reply]
The paths and directories you see in BIIGLE are not the same than you see on your computer. BIIGLE can only work with files that you upload via a storage request. Once these are uploaded, they cannot be changed any more (unless you delete the storage request). If you change something on your computer, this will not be reflected in BIIGLE.

In your uploaded files, I see one folder beneath Skaggerak_SLU that is called xxx.mp4 like the video and in this folder there is the actual video. So if you select the xxx.mp4 folder in the create volume file browser then it should work.

Out of curiosity: How did you create your storage request exactly? I've often seen these full directory trees (C\Users\nicol\...) in storage requests. Normally, you would have to create all these directories manually in the storage request. Did you do this here? Or did you select files with drag and drop, for example, and the directories were automatically created?

**Reply by Nico99-GU** [reply]
The 4R_98.mp4 video you see beneath Skagerrak has already been transformed/used to create my first volume 4R_98SLU. I would like to be able to choose my other mp4 videos. But I canno't upload/drag them into BIIGLE. 
I created the storage request few weeks ago so I am not 100% sure but I think I coppy pasted the directories into BIIGLE then drag and dropped the 4R_98.mp4 video.

**Reply by mzur** [reply]
I can find only one storage request (with one file) for your user account. If you have more videos, please upload them through new storage requests.

**Comment by ohianarevuelta** [comment]
![Photo 2](https://github.com/user-attachments/assets/09d1bf6d-7fc5-4f8c-937e-cbbc94adc524)
![Requested Storage ](https://github.com/user-attachments/assets/99ea3560-b858-42a2-96a5-0031a9085ce8)
Hi,

I requested storage for videos with the path indicated in the attached photo 1; now I am trying to create a new volume, but when I try to upload files, the path is the one I attached in the second photo... How can I upload the files from the storage path I requested? Can I change this path? Could you please help me with this?

Thank you!
Best,
Ohiana

**Reply by mzur** [reply]
Hi Ohiana, you seem to have used a similar but not identical directory structure for different storage requests. Some start at `C` and others at `local` or `Users`. When you create a new volume, select the `C` directory at the top. Then you will be able to navigate to the files of your most recent storage requests.

**Comment by ohianarevuelta** [comment]
Hi,

Ups, easy fix! Massive thanks!!!   I'm truly grateful for your support.

Ohiana

**Comment by ohianarevuelta** [comment]
Dear BIIGLE team,

I'm encountering the following error message when trying to access one of
the videos on the platform:

*"The video file could not be accessed or the codec is not supported by
your browser."*

I believe this may be due to having accidentally deleted the local folder
where the video file was originally located. I have since recreated the
folder with the *same path* and placed the video file back in its original
location, but the issue still persists.

Is there anything else I need to do to re-enable access to the file in
BIIGLE?

I would greatly appreciate any guidance you can provide.

Best regards,


Ohiana

**Reply by mzur** [reply]
Hi Ohiana, there was an issue with the storage backend of BIIGLE today. This is now resolved and the files should load again.

**Comment by ohianarevuelta** [comment]
Thank you! yes it worked with no problem.
